.pylintrc

```diff
@@ -2,14 +2,12 @@
 ignore-paths=^.*/tests/.*$
 disable=
     arguments-renamed,
-    bad-continuation,
     duplicate-code,
     import-error,
     invalid-name,
     logging-format-interpolation,
     logging-fstring-interpolation,
     no-member,
-    no-self-use,
     super-init-not-called,
```

Comment:
Note: the two configs `bad-continuation` and `no-self-use` were triggering the following in pylint 2.15. So I removed them from the .pylintrc and locked the version dependency.
```
************* Module /home/runner/work/screenpy/screenpy/.pylintrc
.pylintrc:1:0: R0022: Useless option value for '--disable', 'bad-continuation' was removed from pylint, see https://github.com/PyCQA/pylint/pull/3571. (useless-option-value)
.pylintrc:1:0: R0022: Useless option value for '--disable', 'no-self-use' was moved to an optional extension, see https://pylint.pycqa.org/en/latest/whatsnew/2/2.14/summary.html#removed-checkers. (useless-option-value)
```

Reply:
Oh, awesome!